### PR TITLE
New hired feedback

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

One DD new hired read our entire doc (yes, it's true) and made some feedback. this PR integrates those feedback to the doc.

### CODE REVIEW

File: content/getting_started/custom_metrics.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Custom Metrics
 kind: documentation
 ---
 
-Datadog allows for custom metrics submission through multiple ways to offer a full understanding of what is happening on your infrastructure.
+Datadog enable custom metrics submission through multiple ways for a comprehensive understanding of what is happening on your infrastructure.
```
Comment (jyee) [inline content/getting_started/custom_metrics.md]: Overall, the old sentence and new changes are awkward. Let's update to:
"Datadog allows you to submit custom metrics in multiple ways in order to provide a comprehensive view of what is happening in your infrastructure"

File: content/getting_started/from_the_query_to_the_graph.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ At each step we comment on the effect of each parameter of the query.
 
 The metric `system.disk.total` (collected by default by the [datadog-agent](/agent)) is seen from different sources.  
 
-First because this metric reported by different hosts, and also because each datadog-agent collects this metric device per device. It adds to metric `system.disk.total` the tag `device:tmpfs` when sending data associated to the disk with the same name, etc.
+First because this metric is reported by different hosts, and also because each datadog-agent collects this metric device per device. It adds to metric `system.disk.total` the tag `device:tmpfs` when sending data associated to the disk with the same name, etc.
```
Comment (jyee) [inline content/getting_started/from_the_query_to_the_graph.md]: This is ok, but let's also remove the first device in "each datadog-agent collects this metric device per device" so it's just "each datadog-agent collects this metric per device"

File: content/getting_started/from_the_query_to_the_graph.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ In this example we consider `host:moby` as having 5 devices. Thus Datadog is sto
 
 Let’s now go over the successive steps followed by our backend for the query presented above.
 
-## Find which timeseries are needed for the query
+## Fetch timeseries are needed for the query
```
Comment (jyee) [inline content/getting_started/from_the_query_to_the_graph.md]: Reject this change. It doesn't make sense.

File: content/getting_started/from_the_query_to_the_graph.md [hunk 4]
```diff
@@ -51,8 +51,8 @@ The idea is then to aggregate data from these sources together to give you a met
 
 [More information about timeseries and tag cardinality](/getting_started/custom_metrics)
 
-Parameter involved: scope
-You can use more than one tag, e.g. {host:moby, device:udev} if you want to data responding to both tags.
+Parameter involved: scope  
+You can use more than one tag, e.g. {host:moby, device:udev} if you want to fetch data responding to both tags.
```
Comment (jyee) [inline content/getting_started/from_the_query_to_the_graph.md]: Line 54 should receive `**` around it to bold it.

File: content/graphing/infrastructure/_index.md [hunk 5]
```diff
@@ -100,9 +100,7 @@ An easy way to accomplish this would be to use the following script that leverag
 
 This script outputs all the current running Agents and their version numbers to a separate document.  Additionally, you can edit the script to input a desired Version number if you would also like all the running Agents that are under a particular version number.  There is also a separate file if you would like to convert the JSON output into a CSV file for your review.
 
-Once you determine which hosts you would like to update you can either manually install the Agent from the [install page](https://app.datadoghq.com/account/settings#agent).   
-
-Or you can make use of one our automation integrations like [Chef](/integrations/chef), [Puppet](/integrations/puppet), or [Ansible](/integrations/ansible).
+Once you determine which hosts you would like to update you can either manually install the Agent from the [install page](https://app.datadoghq.com/account/settings#agent), or you can make use of one of our automation integrations like [Chef](/integrations/chef), [Puppet](/integrations/puppet), or [Ansible](/integrations/ansible).
```
Comment (jyee) [inline content/graphing/infrastructure/_index.md]: s/or you can make use of one of our automation/or you can use one of our automation/

File: content/graphing/notebooks/_index.md [hunk 6]
```diff
@@ -46,11 +46,11 @@ Text in a notebook is formatted with Markdown, which means notebooks can include
 
 ### Manipulating cells
 
-When an existing notebook is opened, its cells are in a closed state. To open a cell for editing, click on it or navigate to it with the arrow keys and press Enter . To close a cell, click outside it or press `⌘+ Enter`. Only one cell may be open at any time.
+When an existing notebook is opened, its cells are in a closed state. To open a cell for editing, click on it or navigate to it with the arrow keys and press Enter . To close a cell, click outside it or press  `⌘+ Enter`. Only one cell may be open at any time.
```
Comment (jyee) [inline content/graphing/notebooks/_index.md]: extra space not needed.

File: content/monitors/monitor_types/forecasts.md [hunk 7]
```diff
@@ -42,7 +42,7 @@ The function has two parameters. The first parameter is for selecting which algo
 {{< img src="monitors/monitor_types/forecasts/query_editor.png" alt="query editor" responsive="true" popup="true" style="width:80%;">}}
 
 ### Forecast Alerts
-In addition to viewing forecasts in dashboards, you can create monitors that trigger when metrics are forecast to reach a threshold. The alert triggers when any part of the range of forecasted values crosses the threshold. The prototypical use case is for monitoring a group of disks with similar usage patterns: `max:system.disk.in_use{service:service_name, device:/data} by {host}`.
+In addition to viewing forecasts in dashboards, you can create monitors that trigger when metrics are forecasted to reach a threshold. The alert triggers when any part of the range of forecasted values crosses the threshold. The prototypical use case is for monitoring a group of disks with similar usage patterns: `max:system.disk.in_use{service:service_name, device:/data} by {host}`.
```
Comment (jyee) [inline content/monitors/monitor_types/forecasts.md]: Technically in English both "forecast" and "forecasted" are acceptable here.

File: content/monitors/monitor_types/metric.md [hunk 8]
```diff
@@ -93,7 +93,7 @@ further_reading:
 7. You can opt to **automatically resolve the monitor from a triggered state**.  
     In general you'll want to leave this option off as you only want an alert to be resolved when it's fixed.  
   
-    This most common use-case for this option is when you have very sparse counters, e.g. for errors. When errors stop occuring the metric stops reporting. This means the monitor don't resolve because there are not anymore values to trigger a resolution. You can also choose a **Recovery thresholds**, those are additional thresholds added to your monitor that indicates an additional condition to a monitor’s recovery from alert or warning states.  
+    The most common use-case for this option is when you have very sparse counters, e.g. for errors. When errors stop occuring the metric stops reporting. This prevents the monitor from resolving because there are no more values to potentially trigger a resolution. You can also set **Recovery thresholds**: thresholds added to your monitor that defines an additional condition to a monitor’s recovery from alert or warning states.
```
Comment (jyee) [inline content/monitors/monitor_types/metric.md]: Remove "potentially" and change "defines" to "define"

File: content/graphing/dashboards/screenboard.md [hunk 9]
```diff
@@ -40,9 +40,9 @@ A user can find all events related to Screenboard changes to the Screenboard the
 
 ## Auditing Dashboards
 
-In dashboards, notifications provide the ability to track changes for audit purposes. Any changes made creates an event in the [event stream](/graphing/event_stream/) that explains the change and displays the user that made the actual change.
+In dashboards, notifications provide the ability to track changes for audit purposes. Any change creates an event in the [event stream](/graphing/event_stream/) that explains the change and displays the user that made the actual change.
 
-If any changes are made to your dashboards, see them with the following event search:
+If any change is made to your dashboards, see them with the following event search:
```
Comment (phrawzty) [inline content/graphing/dashboards/screenboard.md]: Reject; the original sentence is grammatically correct (the proposed change is not).

Also, "see them" is clunky; consider:
> If any changes are made to your dashboards, they can be seen with the following event search:

File: content/monitors/monitor_types/forecasts.md (hunk 7 above)
Comment (phrawzty) [reply content/monitors/monitor_types/forecasts.md]: allow it

**Pull request merged**